### PR TITLE
Homework week 1 Omer Degirmenci

### PR DESCRIPTION
**Pull request by DegirmenciOmer**

### CODE REVIEW

File: Week1/homework/1-helloworld.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+let gr = "Ciao mondo"; //Greek
+console.log(gr);
+
+let ru = "Hola mundo"; //Russian
+console.log(ru);
+
+let po = "Hallo Wereld"; // Polski
+console.log(po);
+
```
Comment (waxs) [inline Week1/homework/1-helloworld.js:30]: Merhaba, nasılsın? Well done, clean syntax and well done on using comments for extra insights on your codebase! 👍

File: Week1/homework/10-compare-arrays.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+//EXERCISE 10 Compare arrays
+
+/*
+Declare 2 variables, that each hold an array. 
+The first array should have 4 items, the second 7 items
+Write a console.log statement that shows the length of each array
+*/
+const WILD_ANIMALS = ["tiger", "elephant", "wolf", "zebra"];
+console.log('The length of the array is '  + WILD_ANIMALS.length + '.');
+const DOMESTIC_ANIMALS = ["dog", "cat", "hen", "horse", "sheep", "cow", "goat"];
```
Comment (waxs) [inline Week1/homework/10-compare-arrays.js:10]: Well done on using meaningful variable names, I see you use all capitals for writing out your `const` variables. 

In most cases this is not necessary, usually I would only assign a number or string to such a const. Reason why is that for instance something should not be able to change.

So `const HOURS_IN_DAY = 24` but an array can despite being a const still receive new values using the `push` method. There are a lot of articles about this subject, for now I would recommend using all capital variable names.

File: Week1/homework/4-log-the-string.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+let myString = 'Omer Degirmenci';
+console.log('the value of myString is \'Omer Degirmenci\'.');
```
Comment (waxs) [inline Week1/homework/4-log-the-string.js:2]: Well done 👍

File: Week1/homework/6-log-an-an-array-of-animals.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+console.log(friends);
+
+//Create a new variable with an array that has 3 of your favorite animals, each in a different string. Make sure the name of the variables says something about what the variable contains.
+let favAnimals = ["cats", "birds", "sheeps"];
+
+//Write a console.log statement that logs the second array.
+console.log(favAnimals);
+
+//Add a statement that adds another string ("Piglet)" to the array of animals.
+favAnimals[3] = "Piglet";
```
Comment (waxs) [inline Week1/homework/6-log-an-an-array-of-animals.js:17]: A better way of achieving this effect would be to use the `push` method. For instance: `favAnimals.push("Piglet");`

File: Week1/homework/8-type-checker.js [hunk 5]
```diff
@@ -0,0 +1,128 @@
+// Declare 4 variables: 2 must be strings and 2 must be objects
+let a = {dessert: 'vla', price: '3$'};
+let b = {chocolate: 'bar', honey: 'jar'};
+let c = 'Amsterdam';
+let d = 'New York';
```
Comment (waxs) [inline Week1/homework/8-type-checker.js:5]: You could have used a more meaningful variable name, let say `d` could have been `string_1` or `city_1` for example.

Comment (waxs) [reply Week1/homework/8-type-checker.js:5]: Just a small note ;)

File: Week1/homework/8-type-checker.js [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+if(y = 21){
+    y = y % 4;
+    console.log(x);
+}
+
+if(z = 13){
+    z = z % 3;
+    console.log(z);
+}
```
Comment (waxs) [inline Week1/homework/8-type-checker.js:87]: Can you explain what happens?

File: Week1/homework/9-log-the-number.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+
+//The remainder operator % gives the remainder of the division of 7 by 3, which gives us 1 as the remainder.
+if(x = 7){
+    x = x % 3;
+    console.log(x);
+}
+
+//The remainder operator % gives the remainder of the division of 21 by 4, which gives us 1 as the remainder.
+if(y = 21){
```
Comment (waxs) [inline Week1/homework/9-log-the-number.js:16]: Well done, forget the other note 👍

**Review by waxs — COMMENTED** [review]
Hi,

Seems like you already picked up some skills! 👍 I made a few notes about how to improve your codebase, hope you'll find them useful. At any time you can reach out on Slack if you feel the need to get some more examples or have questions.

Good luck next week! 

Greets,

Sander

**Review by DegirmenciOmer — COMMENTED** [review]
Review complete.

**Pull request closed without merging**